### ISSUE TITLE
Link on terra-toolkit-boneyard

### ISSUE THREAD

**Issue by ScalableJS** [cerner/terra-toolkit#754]
Link on terra-toolkit-boneyard lead to 404.
Can you update the link or provide new
Thanks

**Comment by benbcai** [comment]
@ScalableJS terra-toolkit has two link references to the archived [terra-toolkit-boneyard](https://github.com/cerner/terra-toolkit-boneyard). Both references have the correct link to terra-toolkit-boneyard. 

[First link reference](https://github.com/cerner/terra-toolkit/blob/f10616700674c53c720ea691b40918b5af0d75c7/packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/upgrade-guides.5/version-1-upgrade-guide.1.tool.md#new-package-structure)
[Second link reference](https://github.com/cerner/terra-toolkit/blob/cb967914cd9f14eeb54d7bf9c1847cc6592d366c/README.md#notice)

Can you clarify which specific link you are accessing that gives you a 404?

**Comment by benbcai** [comment]
@ScalableJS I'm going to close this issue due to lack of activity. Please log a new issue if you have additional questions.

**Comment by ScalableJS** [comment]
![Screenshot_2022-08-02-08-17-09-564_com.github.android.jpg](https://user-images.githubusercontent.com/1230490/182305151-dc39dad6-3c16-4ff7-bc30-0ebec99bbd84.jpg)
